index.js

```diff
@@ -77,4 +113,58 @@ server.post("/users", (req, res) => {
     });
 });
 
+server.delete("/users/:id", (req, res) => {
+  //remove: the remove method accepts an id as it's first parameter
+  //and upon successfully deleting the user from the database
+  //it returns the number of records deleted.
+  //delete = index => h=> [...h.slice(0, index),...h.slice(index+1)]
+  const { id } = req.params;
+  users
+    .remove(id)
+    .then(users => {
+      if (users === 0) {
+        res
+          .status(404)
+          .json({ message: "The user with the specified ID does not exists." });
+      }
+      res.status(201).json(users);
+    })
+    .catch(error => {
```

Comment:
If you want to see what node says is the error, you can also insert `error` (or whatever you called the error parameter) into the json object as a second key/value pair:

```
res.status(500).json({ error: "The user could not be removed", error: error });
```